src/cmake/ExternalAntlr4Cpp.cmake

```diff
@@ -0,0 +1,154 @@
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Was this copy and pasted from somewhere?  Can this be a submodule?  If it cannot be a submodule, please put it in third_party/...

Reply:
Yes, these were copied from https://github.com/antlr/antlr4/tree/master/runtime/Cpp/cmake. I've moved them to `third_party`.